packages/opencensus-web-instrumentation-zone/src/interaction-tracker.ts

```diff
@@ -34,10 +34,13 @@ export type AsyncTask = Task & {
 
 export class InteractionTracker {
   // Allows to track several events triggered by the same user interaction in the right Zone.
-  private shouldCreateNewZone = true;
-  private currentZone: Zone = Zone.root;
+  private currentEventTracingZone?: Zone = undefined;
+
+  // Delay of 50 ms to reset currentEventTracingZone.
+  private readonly RESET_TRACING_ZONE_DELAY: number = 50;
```

Comment:
Nit: can you make this a top level constant rather than a field of the class? You can just use `const RESET_...` at the top of the file